e2e/helpers/testUser.ts

```diff
@@ -84,9 +84,6 @@ export async function loginTestUser(
   await page.getByRole('button', { name: 'Sign in' }).click()
 
   // Ensure that correct username/password were received
-  const loadingButton = page.getByRole('button', { name: 'Logging you in' })
-  await expect(loadingButton).toBeVisible()
-  await expect(loadingButton).not.toBeVisible()
```

Comment:
This state is very short-lived, so was intermittently missed and caused tests to fail, even when wrapped in a Promise.all. This expectation was originally added to help us understand why login intermittently failed, but removing it since actually caused tests to be flakier and the check for the URL and the username/password alert should help us evaluate, if needed